stackcopy_gui.py

```
@@ -1316,34 +1387,34 @@ def _refresh_idle_plan(self) -> None:
             if plan
             else ""
         )
-        generic_output = "Finished stacked photos — renamed with ‘stacked’ added"
+        generic_output = "Finished stacked photos - renamed with ‘stacked’ added"
```

Ruff (0.16.2)

[warning] String contains ambiguous `‘` (LEFT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)